spec/factories/books.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book do
+    genre       { Faker::Book.genre }
+    author      { Faker::Book.author }
+    cover_image { 'image.jpg' }
```

Comment:
Dont use string, use faker for this

Reply:
I used a string since I couldn't find an appropriate Faker field for an image.

Reply:
Is the image it's a file we could fake a file 😉 https://github.com/stympy/faker/blob/master/doc/v1.9.1/file.md

Reply:
Or even better: https://github.com/stympy/faker/blob/master/doc/v1.9.1/avatar.md

Reply:
great! Thanks